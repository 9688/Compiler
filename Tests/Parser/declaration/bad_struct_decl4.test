struct X{
	int x = 5;
};