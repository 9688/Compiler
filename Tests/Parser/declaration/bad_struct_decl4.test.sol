<<<table #1>>>
<tags>
<types>
int
char
double
void
<vars>
var printf type function printf(var  type  to char, ){
} returned void
var scanf type function scanf(var  type  to char, ){
} returned void
var getchar type function getchar(){
} returned int
Ошибка синтаксиса в строке 2, позиции 8: требуется ";"
Ошибка синтаксиса в строке 3, позиции 1: требуется объявление

