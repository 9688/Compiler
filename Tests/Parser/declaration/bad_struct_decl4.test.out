<<<table #1>>>
<tags>
<types>
int
char
double
void
<vars>
Ошибка синтаксиса в строке 2, позиции 8: требуется ";"
Ошибка синтаксиса в строке 3, позиции 1: требуется объявление

